RELEASE-NOTES.md

```diff
@@ -57,6 +57,7 @@
 * `snow streamlit deploy` will check for existing streamlit instance before deploying anything.
 * Fixed `snow git execute` with `/` in name of the branch.
 * `snow app` commands don't enforce ownership of the objects they manage, and rely on RBAC instead.
+* `snow app deploy` and `snow ws deploy` for package entity now allows operating on application packages created outside the CLI
```

Comment:
Maybe we should not mention ws in the release notes, as these are hidden commands.

Reply:
removed